.eslintrc.json

```diff
@@ -18,6 +18,7 @@
   },
   "plugins": ["react"],
   "rules": {
+    "react/react-in-jsx-scope": "off",
```

Comment:
I don't think react/react-in-jsx-scope should be disabled. All the errors related to this have been taken care of in the latest commits. What do you think @Siddhant-K-code ?

Reply:
@faisaljamil25 I think we don't even need react-scope-in-jsx rule while linting in react v17.

Please read the latest new JSX transform documentation mentions the following
"With the new transform, you can use JSX without importing React"
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Reverting and added #75

Reply:
You are correct, we can use JSX without importing React in v17. Thanks for the information.